package.json

```diff
@@ -5,9 +5,9 @@
     "clean": "rm -f *.alfredworkflow; rm -rf output/*",
     "build": "./build.sh",
     "load": "npm run build && open alfred-emoji.alfredworkflow",
-    "test": "tap 'test/**/*.test.js'",
+    "test": "tap 'test/**/*.test.js' --reporter classic",
     "test:watch": "npm test -- --watch",
-    "test-ci": "tap --cov 'test/**/*.test.js'",
+    "test-ci": "tap --cov 'test/**/*.test.js' --reporter classic",
```

Comment:
Why do we need to change the reporter?

Reply:
Oh, I don't know if that's actually necessary for everyone. For *me* when I run it locally without specifying a reporter, I get `Error: Cannot find module './reports/base'`. I found a [tap issue](https://github.com/tapjs/node-tap/issues/624) talking about it.

I have no objection to declaring this a local issue, and removing the explicit specification.

Reply:
Try updating to the latest `tap@14`. Although, that _should_ have installed by default when you ran `npm install`.

Reply:
It's certainly installed at the most-recent tap@14 version:
```sh
 ~/s/alfred-emoji   emojilib-3  npm list                            5.7s  Tue Jun 29 10:36:42 2021
alfred-emoji@1.10.0 /Users/kemayo/src/alfred-emoji
├── emojilib@3.0.2
├── fontkit@1.8.1
├── mock-require@3.0.3
├── pre-commit@1.2.2
├── snazzy@8.0.0
├── standard@12.0.1
├── tap@14.11.0
├── unicode-emoji-json@0.2.1
├── webpack-cli@3.3.12
└── webpack@4.46.0
```

And yet. 🤷🏻

Reply:
Very strange. I can do a fresh clone, `npm install`, and `npm test` without issue. 🤷‍♂️

Reply:
Having looked into it... there's a [different issue](https://github.com/tapjs/node-tap/issues/746) that's this one. It's (apparently) a specific issue with node 16 + tap. Tap would have to be updated to at least v15.0.8 to avoid it.